app/components/works/globus_setup_component.html.erb

```diff
@@ -1,8 +1,17 @@
-<div class="row my-5 mx-2 alert alert-danger">
-  <div class="col-9">
-     Click this button when you have completed setting up your Globus account and we will email you instructions on where to upload your files.
-  </div>
-  <div class="col-3">
-     <%= render Works::GlobusSetupButtonComponent.new(work_version:) %>
-  </div>
-</div>
+<% if work_version.globus_setup_draft? %>
+   <div class="row my-5 mx-2 alert alert-danger">
+      <div class="col-9">
+         Click this button when you have completed setting up your Globus account and we will email you instructions on where to upload your files.
+      </div>
+      <div class="col-3">
+         <%= render Works::GlobusSetupButtonComponent.new(work_version:) %>
+      </div>
+   </div>
+<% else %>
+   <div class="row my-5 mx-2 alert alert-info">
+      <div class="col-12">
+         <p>Upload your files to the Globus folder "<%=work_version.globus_endpoint%>" and then click the "Edit or Deposit" button on this page to complete your deposit.
```

Comment:
When Globus emails the user, and they click the link, do they see the `/uploads/` part of the endpoint? If so, we may want to include that here. If not, I think this is 💯 🆗

Reply:
I do not believe the user sees the `/uploads/` part of the path, but that part of the path should also not be stored in the database and thus should not be shown here either.  See https://github.com/sul-dlss/happy-heron/blob/main/app/jobs/globus_setup_job.rb#L36-L47 for where the path is created and stored in the database

Reply:
Good call!